Task8/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "react_app",
+  "version": "1.0.0",
+  "description": "React-redux app",
+  "main": "index.jsx",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start:dev": "webpack-dev-server --mode development",
+    "build": "webpack",
+    "lint": "stylelint **/*.css --fix",
+    "jsx": "eslint **/*.jsx --fix",
```

Comment:
I don't recommend set `--fix` option as default because usually all such fixes should be reviewed manually.

Reply:
Done